packages/react-divider/src/components/Divider/useDividerClasses.ts

```diff
@@ -0,0 +1,65 @@
+import { makeVariantClasses } from '@fluentui/react-theme-provider';
+import { DividerState, DividerVariants } from './Divider.types';
+
+const GlobalClassNames = {
+  root: 'ms-Divider',
+};
+
+export const useDividerClasses = makeVariantClasses<DividerState, DividerVariants>({
+  name: 'Divider',
+  prefix: '--divider',
+  styles: {
+    root: [
+      GlobalClassNames.root,
+      {
+        alignItems: 'var(--divider-alignContent)',
+        color: 'var(--divider-contentColor)',
+        display: 'flex',
+        margin: '0 10px',
+        padding: '0',
+        position: 'relative',
+        fontFamily: 'var(--divider-fontFamily)',
+        fontSize: 'var(--divider-fontSize)',
+        fontWeight: 'var(--divider-fontWeight)',
+        '::before': {
+          borderTopStyle: 'solid',
+          borderTopColor: 'var(--divider-borderColor)',
+          borderTopWidth: '1px',
+          display: 'flex',
+          content: '""',
+          flexBasis: '100%',
+        },
+        '::after': {
+          borderTopStyle: 'solid',
+          borderTopColor: 'var(--divider-borderColor)',
+          borderTopWidth: '1px',
+          content: '""',
+          display: 'flex',
+          flexBasis: '100%',
+        },
+        _children: {
+          minWidth: 'var(--divider-borderColor)',
+        },
+        children: {
+          minWidth: 'var(--divider-borderColor)',
+        },
```

Comment:
You should only use `_children` and you should use it outside of `root`.

Reply:
no longer applicable